assets/dev/scss/frontend/widgets/spacer.scss

```diff
@@ -0,0 +1,10 @@
+.elementor-spacer-inner {
+	.e-container.e-r & {
```

Comment:
Use an understandable name for the class, here and below

Reply:
Done